tests/util/fake_statsd_server.py

```diff
@@ -179,3 +194,17 @@ def __repr__(self):
         return "<FakeThreadedUDPServer(Packets RX: {}, Metrics RX: {}".format(
             self._payload_counter_shmem_var.value, self._metric_counter_shmem_var.value
         )
+
+
+if __name__ == '__main__':
+    options = {
+      'ignore_timeouts': False,
+      'debug': True,
```

Comment:
Should it be configurable?
```suggestion
      'debug': bool(sys.environ.get("FAKE_STATSD_SERVER_DEBUG", False)),
```

Reply:
> Should it be configurable?

I think having a silent test server would not have a valid use case here since otherwise you wouldn't be able to confirm that the data arrived, that it matches expectations, nor would you see what port/path it's listening on. I think if there's a use case for it later, we can certainly have it as an option it's just that it seemed unnecessary here.